blueocean-events/src/main/java/io/jenkins/blueocean/events/BlueMessageEnricher.java

```diff
@@ -64,7 +66,15 @@
 
     @Override
     public void enrich(@Nonnull Message message) {
+        try {
+            maybeEnrichMessage(message);
+        } catch (Exception e) {
+            LOGGER.log(WARNING, "Unable to enrich message", e);
```

Comment:
Better: `LOGGER.log(WARNING, "Unable to enrich message: " +e.getMessage(), e);`

Reply:
ok - wont' flood the log with long messages? I guess it is ok though, as if this warning happens something is wrong and we do want to know why